app/src/components/trace/AnnotationConfigList.tsx

```diff
@@ -0,0 +1,336 @@
+                gap="size-100"
+              >
+                <TextField
+                  aria-label="Search annotation configs"
+                  value={filter}
+                  onChange={(value) => {
+                    setFilter(value);
+                  }}
+                >
+                  <Input placeholder="Search annotation configs" />
+                </TextField>
+                <DialogTrigger>
+                  <Button aria-label="Create annotation config">New</Button>
+                  <Modal
+                    isDismissable
+                    UNSTABLE_portalContainer={popoverRef ?? undefined}
+                  >
+                    <Dialog>{renderNewAnnotationForm}</Dialog>
+                  </Modal>
+                </DialogTrigger>
+              </Flex>
+            </View>
+            <ListBox
+              css={annotationListBoxCSS}
+              selectionMode="single"
+              selectionBehavior="toggle"
+              aria-label="Annotation Configs"
+              renderEmptyState={() => (
+                <View width="100%" height="100%">
+                  <Flex
+                    direction="column"
+                    alignItems="center"
+                    justifyContent="center"
+                  >
+                    {filter ? (
+                      <Text
+                        style={{
+                          whiteSpace: "pre-wrap",
+                          textAlign: "center",
+                        }}
+                      >
+                        No annotation configs found for &quot;{filter}&quot;
+                      </Text>
+                    ) : (
+                      <Link to="/settings/annotations">
+                        Configure Annotation Configs
+                      </Link>
+                    )}
+                  </Flex>
+                </View>
+              )}
+              onSelectionChange={(keys) => {
+                if (keys === "all" || keys.size === 0) {
+                  return;
+                }
+                const annotationConfigId = keys.values().next().value;
+                toggleAnnotationConfigInProject(annotationConfigId);
+              }}
+            >
+              {filteredAnnotationConfigs.map((config) => (
+                <ListBoxItem
+                  key={config.node.id}
+                  id={config.node.id}
+                  textValue={config.node.name}
+                >
+                  <Flex direction="row" alignItems="center">
+                    <label css={annotationLabelCSS}>
+                      <Flex direction="row" alignItems="center" gap="size-100">
+                        <input
+                          type="checkbox"
+                          checked={annotationConfigIdInProject.has(
+                            config.node.id
+                          )}
+                          readOnly
+                        />
+                        <AnnotationLabel
```

Comment:
just thinking visually we might consider removing the "label" part when showing this in a list. I think while it helps visually correlate things - it's probably cleaner to just have the color swatch + text